webservice/src/services/ApiManager.js

```diff
@@ -144,16 +168,72 @@ export default class ApiManager {
    * @param image
    * @param namespace the namespace where the template should be created
    */
-  createCRDtemplate(
-    course_code,
-    lab_number,
-    description,
-    cpu,
-    memory,
-    image,
-    namespace
-  ) {
-    // TODO: add body
+  createCRDtemplate(namespace, lab_number, description, cpu, memory, image) {
+    const courseName = namespace.split('course-')[1];
+
+    return this.apiCRD.createNamespacedCustomObject(
+      this.templateGroup,
+      this.version,
+      namespace,
+      this.templatePlural,
+      {
+        apiVersion: `${this.templateGroup}/${this.version}`,
+        kind: 'LabTemplate',
+        metadata: {
+          name: `${courseName}-lab${lab_number}`,
+          namespace
+        },
+        spec: {
+          courseName,
+          description,
```

Comment:
As description, I would put the same content as labName (also in the other places where you use description)

Reply:
description is now namespace + labname